examples/single/bundle/src/App.js

```diff
@@ -22,17 +22,30 @@
 import React, { Component } from "react";
 import "regenerator-runtime/runtime";
 
-import { login, getSession, logout, fetch } from "../../../../dist/index";
+import { Session } from "../../../../dist/index";
+import getAuthFetcher from "../../../../dist/dependencies";
 
 class App extends Component {
   constructor(props) {
     super(props);
+    const fetcher = getAuthFetcher({});
+    const session = new Session(
+      {
+        authFetcher: fetcher
+      },
+      "mySession"
+    );
     this.state = {
       status: "loading",
-      loginIssuer: "https://dev.inrupt.net",
-      fetchRoute: "https://jackson.dev.inrupt.net/private",
+      loginIssuer: "https://broker.demo-ess.inrupt.com",
+      fetchRoute:
+        "https://ldp.demo-ess.inrupt.com/116455455448573774513/profile/card#me",
       fetchBody: "",
-      session: null
+      session: session,
+      sessionInfo: {
+        isLoggedIn: session.isLoggedIn,
+        webId: session.webId
```

Comment:
That's a good point. Do you think instead of this duplication a `Session` should have a `readonly` `info` (or `sessionInfo`) attribute ? Manually setting the webid seems like a weird option anyway. In the case of the app here, the two are split because the ref to the session won't change, but the ref to the `sessionInfo` will to indicate a state change.

Reply:
Fixed in 067e7d8d08ee85161d3a581a0a21f61c344678d0